### PR TITLE
Allow different prefixes for cert scopes

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: signingscript/sign.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
 
 # task_cert_type {{{1
-def task_cert_type(task):
+def task_cert_type(context):
     """Extract task certificate type.
 
     Args:
```
Comment (escapewindow) [inline signingscript/sign.py]: Thanks!

Let's update the args in the docstring to match.

Should we specify `project:releng` as the prefix and use it for formats too? That means we'd want to update `signingscript.task.task_signing_formats`.

This changes the public api, so even though we don't have downstream consumers, we're probably looking at a 5.0.0 release for this.

Comment (JohanLorenzo) [reply signingscript/sign.py]: Good idea. I made the changes. While we're at major bumping the version, I also moved `task_cert_type()` to `task.py`

**Comment by JohanLorenzo** [comment]
@tomprince, with the latest changes in this PR, the Thunderbird instance will refuse `project:releng:signing:format:*` but accept `project:comm:thunderbird:releng:signing:format:*`. I haven't deployed it there and it's not urgent. However, I'd like to have it before we go live, just for the sake of consistency.

**Comment by JohanLorenzo** [comment]
This PR is now live in production as [version 5.0.0](https://github.com/mozilla-releng/signingscript/releases/tag/5.0.0). It ran perfectly on Thunderbird: https://tools.taskcluster.net/groups/cJDDKo7CTtCEUPbw5WBXkQ/tasks/AXlJOIGoQIySZov2cL_lTQ/runs/2. It hasn't regressed Firefox: https://tools.taskcluster.net/groups/fyDWM5OWSjGDEv4Is46rgg/tasks/GsNnQYEMRtib1CrcoVZ80Q/runs/0

**Pull request merged**